### PR TITLE
Grants updates

### PR DESCRIPTION
**Pull request by samajammin**
## Description

- Update static templates to show GH edit button
- Link to Grants page from Community page

## Related Issue
None

### CODE REVIEW

**Comment by samajammin** [comment]
@ryancreatescopy @wackerow curious to hear your thoughts - should we make ALL static pages show the GH edit button? Or show that conditionally (e.g. via a frontmatter boolean)? I feel like it's appropriate for the Grants & Community pages. Not sure about Enterprise / all other pages that use the static template.

File: src/templates/static.js [hunk 1]
```diff
@@ -127,16 +127,21 @@ const components = {
...
 }
 
-const StaticPage = ({ data: { mdx } }) => {
+const StaticPage = ({ data: { siteData, mdx }, pageContext }) => {
   const intl = useIntl()
   const isRightToLeft = isLangRightToLeft(intl.locale)
-  const tocItems = mdx.tableOfContents.items
 
-  // TODO some `gitLogLatestDate` are `null` - why?
+  console.log({ mdx })
```
Comment (wackerow) [inline src/templates/static.js:134]: (Intentional?)

**Comment by wackerow** [comment]
@samajammin Agree that ideally it's hidden on enterprise pages. Could make it default to being shown, unless the frontmatter explicitly specifies false.

**Comment by ryancreatescopy** [comment]
I think we can extend to all pages using static template right now. We don't need to approve any PRs for editing enterprise pages if they're controversial changes. But it makes sense that someone should be able to easily fix a typo if they see one.

File: src/templates/static.js (hunk 1 above)
Comment (samajammin) [reply src/templates/static.js:134]: Removing the comment, yes. Adding a console.log, no 😳

Comment (samajammin) [inline src/templates/static.js:134]:
```suggestion
```

**Pull request merged**